### ISSUE TITLE
Multiline expressions/piecewises

### ISSUE THREAD

**Issue by jared-hughes** [DesModder/DesModder#75]
Piecewises like https://www.desmos.com/calculator/rzlwvhxabj are an absolute pain to read, create and edit. It would be great if these could be multiline automatically.

Or, allow typing newlines. Combined with existing indentation through spaces, this would be a great way to improve formatting of long expressions.

Keep in mind https://github.com/mathquill/mathquill/issues/174. It appears that MathQuill does not officially support multiline.

**Comment by jared-hughes** [comment]
One good option that seems to have matrix support: https://github.com/mathquill/mathquill/issues/332 (would likely work if we can use the MathQuill fork)

Possibly, subscripts/superscripts can be abused to half-support multilines. See https://www.desmos.com/calculator/jw0arqc5vh. Adding the following CSS makes the spacing even, but of course it would mess with the appearance of normal subscripts/superscripts

```css
span.dcg-mq-sub {
    font-size: 100%!important;
}

.dcg-mq-supsub {
    margin-bottom: 0!important;
    vertical-align: 0!important;
}
```

Drawback of this method: navigating between the rows is real difficult, and this is not very semantic in the LaTeX

**Comment by jared-hughes** [comment]
Resolved with Text Mode #373
